ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -136,17 +103,23 @@ export default class MessagesList extends Component {
     }
   }
 
-  @action
-  onFilterInputChange(pageFilter) {
-    this.transitionToMessagesWithParams({ pageFilter });
+  @task
+  *handleSearch(evt) {
+    evt.preventDefault();
+    const formData = new FormData(evt.target);
```

Comment:
💡 today I learned about the `FormData` class. Is this why you don't have to manage and track the value on the form inputs in the hbs file?

Reply:
That's right! As long as we pass a `name` to the input, we can access the values by name from the form submit event. When we don't need to show other things based on form values or do validation on keyup, this is a simple way to handle form submits